kubernetes/samples/charts/traefik/README.md

```diff
@@ -4,102 +4,90 @@ This sample demonstrates how to install the Traefik ingress controller to provid
-The chart is in the default repository for Helm.
+This document is based on Traefik version 2.x with the Helm chart at [Traefik Helm Repository](https://github.com/containous/traefik-helm-chart).
+For more information about Traefik, see [Traefik](https://traefik.io/) 
 
 To install the Traefik operator in the `traefik` namespace with default settings:
 ```
-$ helm install --name traefik-operator --namespace traefik stable/traefik
+$ helm repo add traefik https://containous.github.io/traefik-helm-chart
+$ helm repo update
+$ kubectl create namespace traefik
+$ helm install traefik-operator traefik/traefik --namespace traefik
 ```
-Or, with a given `values.yaml`:
+You can also install the Traefik operator, with a custom `values.yaml`. For more detailed information, see [Traefik GitHub Project](https://github.com/containous/traefik-helm-chart/blob/master/traefik/values.yaml).
 ```
-$ helm install --name traefik-operator --namespace traefik --values values.yaml stable/traefik
-```
-With the dashboard enabled, you can access the Traefik dashboard with the URL `http://${HOSTNAME}:30305`, with the HTTP host `traefik.example.com`.
-```
-$ curl -H 'host: traefik.example.com' http://${HOSTNAME}:30305/
-```
-
-## Optionally, download the Traefik Helm chart
-If you want, you can download the Traefik Helm chart and untar it into a local folder:
-```
-$ helm fetch  stable/traefik --untar
-```
-
-## Update the Traefik operator
-After the Traefik operator is installed and running, if you want to change some configurations of the operator, use `helm upgrade` to achieve this.
-```
-$ helm upgrade traefik-operator stable/traefik --values values.yaml 
+$ helm install traefik-operator traefik/traefik --namespace traefik --values values.yaml
 ```
 
 ## Configure Traefik as a load balancer for WLS domains
-In this section we'll demonstrate how to use Traefik to handle traffic to backend WLS domains.
+In this section, we'll demonstrate how to use Traefik to handle traffic to backend WLS domains.
 
 ### 1. Install WLS domains
-Now we need to prepare some domains for Traefik load balancing.
+First, we need to prepare two domains for Traefik load balancing.
 
 Create two WLS domains:
-- One domain with name `domain1` under namespace `default`.
-- One domain with name `domain2` under namespace `test1`.
+- One domain with name `domain1` under namespace `weblogic-domain1`.
+- One domain with name `domain2` under namespace `weblogic-domain2`.
 - Each domain has a web application installed with the URL context `testwebapp`.
 
-### 2. Install the Traefik Ingress
-#### Install a host-routing Ingress
+### 2. Install the Traefik IngressRoute
+#### Install a host-routing IngressRoute
 ```
 $ kubectl create -f samples/host-routing.yaml
 ```
-Now you can send requests to different WLS domains with the unique entry point of Traefik with different hostnames.
+Now you can send requests to different WLS domains with the unique Traefik entry point of different hostnames.
```

Comment:
Don't abbreviate WebLogic

Reply:
you probably need to explain what that file does.  what are these magical hostnames? where did they come from?